aiida/common/log.py

```diff
@@ -201,3 +202,15 @@ def configure_logging(with_orm=False, daemon=False, daemon_log_file=None):
         config['loggers']['aiida']['handlers'].append(handler_dblogger)
 
     dictConfig(config)
+
+
+@contextmanager
+def overide_log_level(level=None):
+    """Temporarily adjust the log-level of logger."""
+    if level is None:
+        level = logging.CRITICAL
```

Comment:
since `logging.CRITICAL` is an `int` and thus immutable, you can directly set it as the default in the function signature, i.e. `def override_log_level(level=logging.CRITICAL)`

Reply:
True, I usually avoid that, because it makes it harder to _explicitly_ use the default argument, but in this case it probably makes sense.